### PR TITLE
YONK-815: API changes to denormalize mobileapps

### PR DESCRIPTION
**Pull request by aamishbaloch**
Changes include updates needed in the APIs as the result of new model changes in [YONK-814](https://github.com/edx-solutions/mobileapps-edx-platform-extensions/pull/15).

### CODE REVIEW

**Comment by aamishbaloch** [comment]
@ziafazal Can you review this?

File: mobileapps/views.py [hunk 1]
```diff
@@ -65,21 +65,18 @@ class MobileAppView(MobileListCreateAPIView):
...
+
         * name: Name of the app,
-        * operating_system: OS version
-            1: Android
-            2: iOS
-            3: Windows
-            4: Other
+        * ios_app_id: ios app ID
+        * android_app_id: Android app ID
+        * ios_download_url: Download URL of the app.
```
Comment (ziafazal) [inline mobileapps/views.py]: typo: iOS download URL of the app

File: mobileapps/views.py [hunk 2]
```diff
@@ -65,21 +65,18 @@ class MobileAppView(MobileListCreateAPIView):
...
+        * android_download_url: Download URL of the app.
```
Comment (ziafazal) [inline mobileapps/views.py]: typo: Anroid download URL of the app

File: mobileapps/views.py [hunk 3]
```diff
@@ -100,21 +97,17 @@ class MobileAppView(MobileListCreateAPIView):
...
-        * identifier: Unique Identifier of the app.
         * name: Name of the app,
-        * operating_system: OS version
-            1: Android
-            2: iOS
-            3: Windows
-            4: Other
+        * ios_app_id: ios app ID
+        * android_app_id: Android app ID
+        * ios_download_url: Download URL of the app.
```
Comment (ziafazal) [inline mobileapps/views.py]: typo: iOS download URL of the app

File: mobileapps/views.py [hunk 4]
```diff
@@ -100,21 +97,17 @@ class MobileAppView(MobileListCreateAPIView):
...
+        * android_download_url: Download URL of the app.
```
Comment (ziafazal) [inline mobileapps/views.py]: typo: Android download URL of the app

File: mobileapps/views.py [hunk 5]
```diff
@@ -132,21 +125,17 @@ class MobileAppView(MobileListCreateAPIView):
...
-        * identifier: Unique Identifier of the app.
         * name: Name of the app,
-        * operating_system: OS version
-            1: Android
-            2: iOS
-            3: Windows
-            4: Other
+        * ios_app_id: ios app ID
+        * android_app_id: Android app ID
+        * ios_download_url: Download URL of the app.
```
Comment (ziafazal) [inline mobileapps/views.py]: typo: iOS download URL of the app

File: mobileapps/views.py [hunk 6]
```diff
@@ -194,21 +183,17 @@ class MobileAppDetailView(MobileRetrieveUpdateAPIView):
...
```
Comment (ziafazal) [inline mobileapps/views.py]: typo: iOS download URL of the app

File: mobileapps/views.py [hunk 7]
```diff
@@ -426,7 +407,7 @@ def post(self, request):
             for mobile_app in mobile_apps:
                 notification_provider = mobile_app.get_notification_provider_name()
                 api_keys = mobile_app.get_api_keys()
-                notification_message = _create_notification_message(mobile_app.identifier, payload)
+                notification_message = _create_notification_message(mobile_app.name, payload)
```
Comment (ziafazal) [inline mobileapps/views.py]: can we user app's id here instead of app name?

**Review by ziafazal — CHANGES_REQUESTED** [review]
@aamishbaloch a few minor tweaks needed.

**Comment by aamishbaloch** [comment]
Done with the changes. @ziafazal Can you review the changes?

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**